pkg/docker/docker_util.go

```diff
@@ -69,7 +69,7 @@ func RemoveSandbox(ctx context.Context, cli Docker, reader io.Reader) error {
 			})
 			return err
 		}
-		return nil
+		return fmt.Errorf("EXIST")
```

Comment:
Define this error as a constant and use it here aswell https://github.com/flyteorg/flytectl/pull/145/files#diff-81ed1d7a8cce59819e2ac00f20daaf9cc8e86219790a191ed190c8c6e22ffed8R83

And lets add a more descriptive error here rathere just EXIST.

Lets calls it SandboxExists